packs/boundary_worker/variables.hcl

```diff
@@ -0,0 +1,347 @@
+variable "app_boundary_helper_path" {
+  type        = string
+  description = "Path to the Boundary Helper binary."
+  default     = "boundary-helper"
+}
+
+variable "app_boundary_helper_output_file_mode" {
+  type        = string
+  description = "File Mode of the Output File created by the Boundary Helper binary."
+  default     = "0644"
+}
+
+variable "app_cors_enabled" {
+  type        = bool
+  description = "Toggle to enable CORS support for the Boundary Worker."
+  default     = true
+}
+
+variable "app_cors_allowed_origins" {
+  type        = list(string)
+  description = "Allowed CORS Origins for the Boundary Worker."
+
+  default = [
+    "*",
+  ]
+}
+
+variable "app_disable_mlock" {
+  type        = bool
+  description = "Toggle to disable MLock for the Boundary Worker."
+  default     = true
+}
+
+variable "app_enable_hcp_boundary_support" {
+  type        = bool
+  description = "Toggle to enable HCP Boundary Support (and forego self-hosted Boundary Enterprise Cluster registration workflows."
+  default     = true
+}
+
+# full worker name format is `<prefix>-<NOMAD_SHORT_ALLOC_ID>-<NOMAD_ALLOC_INDEX>`
+# value will be transformed to lowercase
+variable "app_worker_name_prefix" {
+  type        = string
+  description = "Prefix for the Boundary Worker Name."
+
+  # a trailing separation character (`-`) will be added automatically to allow
+  # for easier identification of Nomad-orchestrated Workers in the Boundary UI
+  default = "nomad"
+}
+
+variable "app_worker_description" {
+  type        = string
+  description = "Description for the Boundary Worker."
+  default     = "Nomad-managed Boundary Worker."
+}
+
+# only relevant in a non-HCP Boundary setting
+variable "app_initial_upstreams" {
+  type        = list(string)
+  description = "Initial Upstreams for the Boundary Worker."
+  default     = []
+}
+
+variable "app_tls_disable" {
+  type        = bool
+  description = "Toggle to disable TLS support for the Boundary Worker."
+  default     = true
+}
+
+variable "app_tls_cert_file" {
+  type        = string
+  description = "Specifies the path to the certificate for the Boundary Worker."
+  default     = ""
+}
+
+variable "app_tls_key_file" {
+  type        = string
+  description = "Specifies the path to the private key for the certificate for the Boundary Worker."
+  default     = ""
+}
+
+variable "app_tls_min_version" {
+  type        = string
+  description = "Specifies the minimum supported TLS version for the Boundary Worker."
+  default     = "tls12"
+}
+
+variable "app_tls_max_version" {
+  type        = string
+  description = "Specifies the maximum supported TLS version for the Boundary Worker."
+  default     = "tls13"
+}
+
+variable "app_tls_cipher_suites" {
+  type        = string
+  description = "Overridden List of supported ciphersuites for the Boundary Worker."
```

Comment:
```suggestion
  description = "Overridden List of supported ciphersuites for the Boundary Worker (only applies for TLS1.2 and older, but not on TLS1.3 which is the default)."
```

Reply:
I changed this to something else, but kept it functionally intact